frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -466,6 +499,20 @@ export function FieldValuesWidgetInner({
             optionRenderer={optionRenderer}
             checkedColor={checkedColor}
           />
+        ) : !isSimpleInput ? (
+          <MultiAutocomplete
+            onSearchChange={onInputChange}
+            onChange={onChange}
+            value={value.filter(v => v !== null && v !== undefined)}
+            data={options.map(renderStringOption)}
+            renderValue={value => optionRenderer?.([value])}
+            placeholder={tokenFieldPlaceholder}
+            shouldCreate={shouldCreate}
+            autoFocus={autoFocus}
+            prefix={prefix}
+            parseValue={parseFreeformValue}
+            maxSelectedValues={multi ? undefined : 1}
```

Comment:
We shouldn't use this component for single value selection. It doesn't work great for this case. Please use a `TextInput` or a similar component instead.